src/workerd/server/tests/python/jspi/worker.py

```diff
@@ -0,0 +1,23 @@
+from asyncio import sleep
+
+from pyodide import __version__
+
+
+def jspi_sleep():
+    if __version__ == "0.26.0a2":
+        print("JSPI not supported on 0.26.0a2, skipping")
+        return
```

Comment:
You skip it already via the `skip_python_flags`, may be good to remove this so that it doesn't silently pass if we do want to run it against 0.26.0 in the future.

Reply:
It won't ever work against 0.26.0

Reply:
oh, duh, I forgot about dev. Makes sense, ignore me.